react/states_and_effects/project_cv_application.md

```diff
@@ -70,6 +70,6 @@ Similar to Netlify, Vercel also offers a few ways to deploy, but we will be impo
 
 #### Cloudflare Links
 
-- [Homepage](https://pages.cloudflare.com/)
-- [Documentation Website](https://developers.cloudflare.com/pages)
-- This [section in the Cloudflare docs](https://developers.cloudflare.com/pages/framework-guides/deploy-a-vite3-project/) discusses these very steps
+- [Cloudflare Homepage](https://pages.cloudflare.com/)
+- [Cloudflare Documentation Website](https://developers.cloudflare.com/pages)
+- [Vite deployment to Cloudflare](https://developers.cloudflare.com/pages/framework-guides/deploy-a-vite3-project/) discusses these very steps
```

Comment:
```suggestion
- [Cloudflare Pages Homepage](https://pages.cloudflare.com/)
- [Cloudflare Pages documentation website](https://developers.cloudflare.com/pages)
- [Vite deployment to Cloudflare Pages](https://developers.cloudflare.com/pages/framework-guides/deploy-a-vite3-project/) discusses these very steps
```
We need to specify `Pages` as `Cloudflare Pages` is the name of the service (which is a separate service to `Cloudflare` on its own).